certified-connectors/Secib/apiDefinition.swagger.json

```diff
@@ -1018,6 +997,209 @@
+                  "title": "HttpStatusCode"
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Internal Server Error.",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "message": {
+                  "type": "string",
+                  "description": "Error message.",
+                  "title": "message"
+                },
+                "statusCode": {
+                  "type": "integer",
+                  "description": "Http status code of the request.",
+                  "title": "HttpStatusCode"
+                }
+              }
+            }
+          },
+          "503": {
+            "description": "Service Unavailable.",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "message": {
+                  "type": "string",
+                  "description": "Error message.",
+                  "title": "message"
+                },
+                "statusCode": {
+                  "type": "integer",
+                  "description": "Http status code of the request.",
+                  "title": "HttpStatusCode"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Retrieving view's data with parameters",
+        "tags": ["Views"],
+        "description": "Retrieve data from a view in the SECIB database, using a where clause.",
+        "operationId": "GetViewByViewNameV2",
+        "parameters": [
+          {
+            "name": "name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Name of the view in the SECIB database.",
+            "x-ms-summary": "View's name",
+            "x-ms-visibility": "important",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "cabinet",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Identifier of the firm to work with.",
+            "x-ms-summary": "Firm's identifier",
+            "x-ms-dynamic-values": {
+              "operationId": "GetCabinets",
+              "value-path": "id",
+              "value-collection": "result/items",
+              "value-title": "nom"
+            }
+          },
+          {
+            "name": "columnName",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Name of the view's column which will be used to order the result.",
+            "x-ms-summary": "Order by column's name"
+          },
+          {
+            "name": "filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The where clause that will be used in the sql query (e.g. : name LIKE '%Power Automate%').",
```

Comment:
@matthieuHellaSecib parameter descriptions can't contain Power Platform product names.
Please change the example for LIKE clause.
More information [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions).